components/scheduler/grouping.md

```diff
@@ -0,0 +1,573 @@
+---
+title: Grouping
```

Comment:
I recommend you add to the url and various title the keyword "resource grouping"